docs/provisioning.md

```diff
@@ -0,0 +1,111 @@
+# Provisioning the IDR
+
+## Overview of the architecture
+
+The IDR contains two main groups of servers:
+
+The production (public-facing) IDR (3 servers):
+- Database
+- OMERO.server
+- Nginx gateway
+
+The virtual analysis environment (VAE) IDR (3 servers):
+- Database
+- OMERO.server
+- Docker Manager
+
+
+## Ansible prerequisites
+
+Almost all of the provisioning and deployment in the IDR is done using Ansible 2.1.
+All ansible commands should be run from a shell in the [`ansible`](../ansible) directory.
+
```

Comment:
It could be confusing, depending on whether you see the root or the docs dir as your base.